src/server/controllers/getCount.js

```diff
@@ -7,7 +8,20 @@ exports.get = async (req, res) => {
     const countParticipant = await participant.findAndCountAll();
     const countCourse = await course.findAndCountAll();
     const countWorker = await worker.findAndCountAll();
-    res.send({ countParticipant, countCourse, countWorker });
+    const employed = 'employed';
+    const countEmployedParticipant = await participant.findAndCountAll({
+      where: {
+        employment_outcomes: {
+          [Op.like]: `${employed}`,
```

Comment:
I think like is not important in this case
you can convert it like this
<code>employment_outcomes: employed</code>

Reply:
ok @mohannadhanafi :+1: